setup.py

```diff
@@ -63,7 +63,7 @@ def run(self):
     "humanize>=0.5.1",
     "ruamel.yaml>=0.16.1",
     "funcy>=1.12",
-    "pathspec>=0.5.9",
+    "pathspec@git+https://github.com/cpburnz/python-path-specification@master",
```

Comment:
let's specify a specific commit and not `master`.

Reply:
@efiop, got it

Reply:
@efiop , changed it